.github/workflows/pre-release.yml

```
@@ -0,0 +1,312 @@
+name: Prepare CDN Pre-Release and NPM Publish
```

// Small grammar nit.  Sounds like you are preparing the NPM publish, as opposed to actually doing it
```suggestion
name: Release - Step 1: Prepare CDN Pre-Release, Publish NPM
```